### ISSUE TITLE
Issue with spacing

### ISSUE THREAD

**Issue by jainpranav1** [wokwi/hdl-parser#1]
Great parser! This tool helped me to make this HDL Visualizer: https://marketplace.visualstudio.com/items?itemName=PranavJain.nand2tetris-hdl-visualizer&ssr=false#overview. One issue with this parser is that if there is a space around the equal sign, it throws an error.

This is discussed in detail in this issue: https://github.com/jainpranav1/Nand2Tetris-HDL-Visualizer/issues/3.  Could you provide a fix?

**Comment by mightyhorst** [comment]
If I leave a space between the equals sign it will throw an error. e.g. `a=nota` works but `a = nota` with spaces throws a syntax error

### SyntaxError
This code will throw a regex error

`SyntaxError: Expected "false", "true", or [a-z,A-Z] but " " found.`

```
CHIP Or {
    IN a, b;
    OUT out;

    PARTS:
    Nand (a = nota, b=notb, out=out);
    Not (in=a, out=nota);
    Not (in=b, out=notb);
}
```

Note the whitespace in the line `a = nota`

**Comment by urish** [comment]
Thanks for reporting, fix released as 1.0.2

**Comment by jainpranav1** [comment]
Thank you so much!
